vault/provider.go

```diff
@@ -27,6 +27,12 @@ const (
 	// versions of Vault.
 	// We aim to deprecate items in this category.
 	UnknownPath = "unknown"
+
+	// DefaultMaxHTTPRetries is used for configuring the api.Client's MaxRetries.
+	DefaultMaxHTTPRetries = 2
+
+	// MaxGetRetriesMultiplier is used
```

Comment:
Why a multipler?

Reply:
If we think that the default or whatever is provided by the user is good enough, then I can drop it. The docs claim the retry is used for 5xx type errors.

Reply:
Dropped the multiplier in https://github.com/hashicorp/terraform-provider-vault/pull/1263/commits/d70cc2618e778a4ef3f54b8549c0eedc3999cdd1